production/tanka/grafana-agent/config.libsonnet

```diff
@@ -77,9 +77,17 @@
       },
     },
     deployment_agent_config: self.agent_config {
-      configs: std.map(function(c) c {
-        host_filter: false,
-      }, super.configs),
+      prometheus+: {
+        configs: [{
+          name: 'agent',
```

Comment:
Should this be `agent-deployment`?

Reply:
I don't think it matters much but it's a good idea to change it just to be safe.